app/services/planning_application_creation.rb

```diff
@@ -0,0 +1,81 @@
+class PlanningApplicationCreation
+  class PlanningApplicationCreationInvalidProperty < StandardError; end
+
+  def initialize(**params)
+    @local_authority = params.fetch(:local_authority, nil)
+
+    @reference = params.fetch(:reference, nil)
+    @area = params.fetch(:area, nil)
+    @proposal_details = params.fetch(:proposal_details, nil)
+    @received_at = params.fetch(:received_at, nil)
+    @officer_name = params.fetch(:officer_name, nil)
+    @decision = params.fetch(:decision, nil)
+    @decision_issued_at = params.fetch(:decision_issued_at, nil)
+    @view_documents = params.fetch(:view_documents, nil)
+
+    @uprn = params.fetch(:uprn, nil)
+    @property_code = params.fetch(:property_code, nil)
+    @property_type = params.fetch(:property_type, nil)
+
+    @full = params.fetch(:full, nil)
+    @address = params.fetch(:address, nil)
+    @town = params.fetch(:town, nil)
+    @postcode = params.fetch(:postcode, nil)
+    @map_east = params.fetch(:map_east, nil)
+    @map_north = params.fetch(:map_north, nil)
+    @ward_code = params.fetch(:ward_code, nil)
+    @ward_name = params.fetch(:ward_name, nil)
+  end
+
+  def perform
+    importer
```

Comment:
You could just call `#importer` `#perform` instead?

Of alternatively use `#alias_method` here? https://apidock.com/ruby/Module/alias_method

Reply:
The `#alias_method` is not going to work as `#importer` is private. I defined a second public method `#perform` to call it!